lib/dul_arclight/traject/ead2_config.rb

```diff
@@ -600,15 +625,26 @@
     accumulator.replace range.years
   end
 
+  to_field 'date_range_sim', extract_xpath('./did/unittitle/unitdate') do |_record, accumulator, context|
+    unless context.output_hash['date_range_sim']
+      clean_range = accumulator.map { |v| v.gsub(/([^()]*)\(.*\)/, '\1').tr('^0-9\\-/ ', '').strip }
+                        .select { |date| date.match? /^\d\d\d\d[-\/]\d\d\d\d$/}
+      # logger.debug accumulator unless clean_range.empty?
+      # logger.debug clean_range unless clean_range.empty?
+      range = Arclight::YearRange.new(clean_range.include?('/') ? clean_range : clean_range.map { |v| v.tr('-', '/') })
+      accumulator.replace range.years
+    end
+  end
+
   NAME_ELEMENTS.map do |selector|
     to_field 'names_ssim', extract_xpath("./controlaccess/#{selector}"), unique, strip
     to_field "#{selector}_ssm", extract_xpath(".//#{selector}"), unique, strip
   end
 
   # DUL CUSTOMIZATION: Bugfix field geogname --> places
-  to_field 'places_sim', extract_xpath('./controlaccess/geogname')
-  to_field 'places_ssm', extract_xpath('./controlaccess/geogname')
-  to_field 'places_ssim', extract_xpath('./controlaccess/geogname')
+  to_field 'places_sim', extract_xpath('./controlaccess/geogname|./controlaccess/controlaccess/geogname')
+  to_field 'places_ssm', extract_xpath('./controlaccess/geogname|./controlaccess/controlaccess/geogname')
+  to_field 'places_ssim', extract_xpath('./controlaccess/geogname|./controlaccess/controlaccess/geogname')
```

Comment:
These would probably be better served in the solr schema with `copyField` directives instead of repeated code. Barring that, I wonder if if should just be, e.g., 

```ruby 
to_field 'places_ssm' do |_rec, acc, context|
  acc.replace context.output_hash.fetch("places_sim", [])
end
```

We should really have a standard macro for this, e.g.

```ruby
def same_as(field, default: [])
  ->(_rec, acc, context) { acc.replace context.output_hash.fetch(field, default)}
end
```